index.html

```diff
@@ -434,11 +434,117 @@ <h2 id="protocol-registry">
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
     </p>
+    <ol>
+      <li>MUST be standardized at a <a href=
+      "https://www.w3.org/liaisons/">consortium the W3C liaises with</a>
+      </li>
+      <li>MUST have a stable URL that points to a publicly available
+      specification.
+      </li>
+      <li>MUST define a representation, as either a [[WebIDL]] [=dictionary=]
+      or a JSON object, of the [=digital credential/exchange protocol=] request
+      structure (i.e., the [=dictionary=] to which the
+      {{DigitalCredentialsProvider}}'s {{DigitalCredentialsProvider/request}}
+      is [=converted to idl values|converted=] to before it is passed onto
+      underlying platform).
+      </li>
+      <li>MUST define a representation, as either a [[WebIDL]] [=dictionary=]
+      or a JSON object, of the [=digital credential/exchange protocol=]
+      response structure (i.e., the [=dictionary=] to which the
+      {{DigitalCredential}}'s {{DigitalCredential/data}} is [=converted to idl
+      values|converted=] before it is made available to the relying party).
+      </li>
+      <li>MUST define validation rules for members of the request and response
+      structures.
+      </li>
+      <li>MUST have undergone privacy review by the W3C's Privacy Interest
+      Group and Federated Identity Working Group.
+      </li>
+      <li>MUST have undergone security review by the Federated Identity Working
+      Group.
+      </li>
+      <li>MUST have implementation commitment from at least two implementers in
+      independent browser engines, to meet the W3C's adequate implementation
+      experience requirements.
+      </li>
+      <li>MUST be defined in a specification which is available publicly at the
+      URL listed in the registry.
+      </li>
+      <li>MUST have formally recorded consensus by the Working Group to be
+      included in the registry.
+      </li>
+    </ol>
+    <h4>
+      Presentation-specific inclusion criteria
+    </h4>
+    <p>
+      To be included as a presentation protocol in the registry (used with
+      `navigator.credentials.get`), the [=digital credential/exchange
+      protocol=]:
+    </p>
+    <ol>
+      <li>MUST support response encryption.
+      </li>
+      <li>MUST encrypt any response containing personally identifiable
+      information (PII).
+      </li>
+    </ol>
+    <h3>
+      Change process
+    </h3>
+    <p>
+      To add a new [=digital credential/exchange protocol=] to the registry, or
+      to update an existing one:
+    </p>
+    <dl>
+      <dt>
+        Define a protocol identifier
+      </dt>
+      <dd>
+        The protocol identifier MUST be a unique string that is not already in
+        use in the registry. Use only lowercase ASCII letters, digits, and
+        hyphens (e.g., "protocol", "the-protocol"). Avoid using version numbers
+        in the protocol identifier. The protocol identifier MUST uniquely
```

Comment:
2025-04-11 WG call: remove ` Avoid using version number in the protocol identifier.`

Reply:
Note that this avoidance results in a requirement that the protocol include version negotiation, even if that is as simple as the two sides of a protocol exchange announcing their version, each to the other, with the potential that either or both sides will respond, `I cannot interact with that version of this protocol. <Something> must be updated.`